src/contracts/Value/ItemGroupInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\Personalization\Value;
+
+interface ItemGroupInterface
+{
+    public function getGroupIdentifier(): string;
```

Comment:
Doesn't it make sense to name it `getIdentifier` instead? Identifiers are a concept that we're using in multiple objects and it's unlikely for a group to have a second, different identifier.

```suggestion
    public function getIdentifier(): string;
```